user/troubleshooting/resume-suspend-troubleshooting.md

```diff
@@ -9,6 +9,8 @@ ref: 94
 title: Suspend/Resume Troubleshooting
 ---
 
+First check if there are any remarks for similar devices in the [Hardware Compatibility List (HCL)](https://www.qubes-os.org/hcl/).
```

Comment:
Please always use relative links for internal website links (see [here](https://www.qubes-os.org/doc/doc-guidelines/#markdown-conventions) for more information).

Reply:
Changed to `/hcl/` and according to [conventions](https://www.qubes-os.org/doc/doc-guidelines/#markdown-conventions) rewrapped to max line lenght 80.